app/main/controllers/activity/RTMediaBuddyPressActivity.php

```diff
@@ -847,4 +848,26 @@ public function rtm_bp_activity_permalink_access( $args ) {
 		return $has_access;
 
 	}
+
+	public function rtm_check_privacy_for_comments( $comment_id, $r, $activity ) {
+		global $wpdb;
+		$db_prefix = $wpdb->base_prefix;
```

Comment:
Are we going to need base sites prefix? what if the site is multisite and using different prefix than the base site
may check this function `$wpdb->get_blog_prefix();`

Also please align all variable declarations properly